src/tools/web/client/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Daniel Bugl
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
```

Comment:
Even though MIT actually is similar permissive as BSD, you should explicitely note somewhere that the code is dual licensed (or BSD-only, then simply remove this file). Otherwise users of Elektra would need to copy multiple of license files.